code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -225,3 +225,39 @@
 	icon_state = "cyborg_upgrade2"
 	required_module = list(/obj/item/weapon/robot_module/butler)
 	modules_to_add = list(/obj/item/weapon/reagent_containers/glass/beaker/large/cyborg,/obj/item/weapon/kitchen/utensil/knife/large,/obj/item/weapon/storage/bag/food/borg)
+
+//Grippers. All there so admins can bus them if they want/we can actually testmerge on nexis serbia.
+/obj/item/borg/upgrade/magnetic_gripper
+	name = "service gripper module board"
+	desc = "Used to give a service cyborg an service gripper."
+	icon_state = "cyborg_upgrade2"
+	required_module = list(/obj/item/weapon/robot_module/butler)
+	modules_to_add = list(/obj/item/weapon/gripper/service)
+
+/obj/item/borg/upgrade/magnetic_gripper
+	name = "magnetic gripper module board"
+	desc = "Used to give a engineering cyborg an magnetic gripper."
+	icon_state = "cyborg_upgrade2"
+	required_module = list(/obj/item/weapon/robot_module/engineering)
+	modules_to_add = list(/obj/item/weapon/gripper/no_use/magnetic)
+
+/obj/item/borg/upgrade/magnetic_gripper
```

Comment:
You've got overwriting subtypes, the only one that exists is the sheet insertion gripper upgrade